### PR TITLE
Quantum diffusion

### PR DESCRIPTION
**Pull request by willrogers**
Let's open a pull request and let Travis run the tests @carmignani

### CODE REVIEW

**Comment by willrogers** [comment]
It says `LINK : error LNK2001: unresolved external symbol initStrMPoleSymplectic4QuantPass`. I'll check to see if it's different to the other integrators.

File: atintegrators/StrMPoleSymplectic4QuantPass.c [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+    }
+    StrMPoleSymplectic4QuantPass(r_in,Elem->Length,Elem->PolynomA,Elem->PolynomB,
+            Elem->MaxOrder,Elem->NumIntSteps,Elem->FringeQuadEntrance,
+            Elem->FringeQuadExit,Elem->fringeIntM0,Elem->fringeIntP0,
+            Elem->T1,Elem->T2,Elem->R1,Elem->R2,
+            Elem->RApertures,Elem->EApertures,Elem->KickAngle,Elem->Energy,num_particles);
+    return Elem;
+}
+
+MODULE_DEF(StrMPoleSymplectic4RadPass)        /* Dummy module initialisation */
```
Comment (willrogers) [inline atintegrators/StrMPoleSymplectic4QuantPass.c]: It should be `MODULE_DEF(StrMPoleSymplectic4QuantPass)` here.

**Comment by willrogers** [comment]
It looks like there was a problem with the online Windows build. Anyway, I think it's passing now.

**Comment by carmignani** [comment]
I wait to merge, because we found some strange behavior in some simulation results. There are some photons emitted at very high energy and probably it's a bug.

**Comment by carmignani** [comment]
Bug fixed, so I merge the pull request.

**Pull request merged**